README.md

```diff
@@ -50,13 +50,14 @@ For now, we have the following resources available:
 1. [Angular guidelines](/guides/development/angular-guidelines.md)
 1. [Angular 17 guidelines](/guides/development/angular-17-guidelines.md)
 1. [React guidelines](/guides/development/react-guidelines.md)
-2. [Back-end guidelines](/guides/development/back-end-development-guidelines.md)
-3. [Code reviews guidelines](/guides/development/code-reviews-guidelines.md)
-4. [Coding guidelines](/guides/development/coding-guidelines.md)
-5. [Ruby & Rails guidelines](/guides/development/ruby-guidelines.md)
-6. [WIP: Testing guidelines](/guides/development/testing-guidelines.md)
+1. [Back-end guidelines](/guides/development/back-end-development-guidelines.md)
+1. [Code reviews guidelines](/guides/development/code-reviews-guidelines.md)
+1. [Coding guidelines](/guides/development/coding-guidelines.md)
+1. [Ruby & Rails guidelines](/guides/development/ruby-guidelines.md)
+1. [WIP: Testing guidelines](/guides/development/testing-guidelines.md)
+1. [WIP: Security guidelines](/guides/security/README.md)
```

Comment:
The security guides will always be a work-in-progress because we will continually add more content. However, in its current state, it is already useful and complete, so we can remove the WIP tag.

Additionally, I suggest moving the Security Guidelines to the first item in the `Our Guides` section. It is not only relevant to development but also contains important information for managers and other roles. It is significant enough to be placed in the first position.